index.html

```diff
@@ -9,47 +9,154 @@
     
 </head>
 <body>
-	<header class="header">
-		<div  class="logo"></div> 
-      <nav class="menu" id="menu">
-            
-        <div class="menu-close">
-            <a href="#" class="menu-close-icon"><img src="static/ic_menu.svg" alt="" srcset=""></a>
-        </div>
-        </nav>
-
-    </header>
-
-	<main>
-        <section class="bio">
-            
-            <h1 class="bio-header">
-                Bleu Yves <br> Sopoude<br>
-            
-            </h1>
-
-            <p class="bio-info">  
-            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br>  <br>
-
-            If you like what you see and have a project you need coded, don’t hesitate to contact me.
-            </p>
-
-			
-			<div class="link">
-                <p><button class="link-text" >Start collaboration</button></p>	
-            </div>
-                    
-            <div class="image"></div>
-                   	
-            
-        </section>
-
-
-    </main>
-
-	
-	
-</body>
+	<section class="work">
+        <h2 class="projects">Projects</h2>
+        <div class="proj-1">
+          <img src="static/Proj1.png" alt="project 1" />
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
```

Comment:
- [ ] This icon should align to the center in the cross-axis and should NOT have a border around it. Kindly fix this.

Your work
![image](https://user-images.githubusercontent.com/84156714/236498981-688eb9d4-8df4-4da5-b96f-3c2283cab8de.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/236499103-ed8c6647-37e6-476a-9196-016b2eeaa794.png)